src/Voron/Data/Fixed/FixedSizeTree.cs

```diff
@@ -1653,7 +1653,7 @@ public override string ToString()
             return Name.ToString();
         }
 
-        public long GetNumberOfEntriesAfter(TVal value, out long totalCount)
+        public long GetNumberOfEntriesAfter(TVal value, out long totalCount, Stopwatch overallDuration)
```

Comment:
My concern is that now this can return inconsistent results since the method signature doesn't indicate that it can be estimated. I'm worried about someone using it for different purpose in the future and it will work fine during development but not in production.

Reply:
This _does_ return wrong results, but the nice thing here is that it fixed itself, since each time it loads more of the relevant pieces to memory.

Do you have a good idea on how we can handle that? I can make it return an object that says `Estimated`, of course. But not sure how useful that would be.